test/bid.spec.js

```diff
@@ -916,4 +915,213 @@ describe('Bid bidObject', function() {
             done();
         });
     });
+
+    describe('#getPbBid', function() {
+
+        it('is a function', function(done) {
+            bidObject.getPbBid.should.be.a('function')
+            done();
+        });
+
+        it('returns gross cpm', function(done) {
+            expect(bidObject.getPbBid()).to.be.equal(undefined);
+            done();
+        });
+    });
+
+    describe('#setPbBid', function() {
+
+        it('is a function', function(done) {
+            bidObject.setPbBid.should.be.a('function')
+            done();
+        });
+
+        it('should set prebid bid in pBBid', function(done) {
+            expect(bidObject.pbbid).to.equal(undefined);
+            bidObject.setPbBid({"test":"bid"}).should.deep.equal(bidObject);
+            expect(bidObject.pbbid).to.deep.equal({"test":"bid"});
+            done();
+        });
+    });
+
+    describe('#getVastXml', function() {
+
+        it('is a function', function(done) {
+            bidObject.getVastXml.should.be.a('function')
+            done();
+        });
+
+        it('returns vast xml', function(done) {
+            expect(bidObject.getVastXml()).to.be.equal(undefined);
+            done();
+        });
+    });
+
+    describe('#setVastXml', function() {
+
+        it('is a function', function(done) {
+            bidObject.setVastXml.should.be.a('function')
+            done();
+        });
+
+        it('should set vast xml', function(done) {
+            expect(bidObject.vastXml).to.equal(undefined);
+            bidObject.setVastXml("").should.deep.equal(bidObject);
+            expect(bidObject.vastXml).to.deep.equal("");
+            done();
+        });
+    });
+
+    describe('#getvastUrl', function() {
+
+        it('is a function', function(done) {
+            bidObject.getVastUrl.should.be.a('function')
+            done();
+        });
+
+        it('returns vast xml', function(done) {
+            expect(bidObject.getVastUrl()).to.be.equal(undefined);
+            done();
+        });
+    });
+
+    describe('#setVastUrl', function() {
+
+        it('is a function', function(done) {
+            bidObject.setVastUrl.should.be.a('function')
+            done();
+        });
+
+        it('should set vast url', function(done) {
+            expect(bidObject.vastUrl).to.equal(undefined);
+            bidObject.setVastUrl("").should.deep.equal(bidObject);
```

Comment:
rather than using an empty string as input try some randomly generated string as input, this will be a better case to check whether the function is not returning some hard-coded value, the same review applies to other such tests

Reply:
Mae sense, 
generated random string in setter functions and checking for same value.